plugins/CoreAdminHome/templates/trackingCodeGenerator.twig

```diff
@@ -142,6 +142,19 @@
                     </div>
                 </div>
 
+                {# cross domain support #}
+                <div id="jsCrossDomain" class="inline-help-node">
+                    {{ "CoreAdminHome_JSTracking_CrossDomain"|translate }}
+                </div>
+
+                <div piwik-field uicontrol="checkbox" name="javascript-tracking-cross-domain"
+                     ng-model="jsTrackingCode.crossDomain"
+                     ng-change="jsTrackingCode.updateTrackingCode();"
+                     disabled="jsTrackingCode.isLoading || jsTrackingCode.siteUrls.length <= 1"
```

Comment:
`|| jsTrackingCode.siteUrls.length <= 1` is not working somehow, the option is always enabled even when there is only one Alias URL defined. Maybe you manage to fix this @tsteur